docs/RELEASE-NOTES.rst

```diff
@@ -4,6 +4,26 @@ Release Notes for BIG-IP Controller for Kubernetes
 |release|
 ---------
 
+Added Functionality
+```````````````````
+
+* Watch all nodes by default; watch a subset of nodes with a user-specified label.
+* Create BIG-IP SSL Profiles from Kubernetes Secrets via Ingress TLS.
+* Create BIG-IP objects from OpenShift Route resources.
+  - This includes unsecured, edge, passthrough, and re-encrypt Routes.
+
+* This is a feature complete upgrade from the OpenShift F5Router. 
+  See `[Replace the OpenShift F5 Router with the BIG-IP Controller] <http://clouddocs.f5.com/containers/elbert/openshift/replace-f5-router.html>`_ 
+  for more information.
+
+Bug Fixes
+`````````
+* Properly configures http redirect rules on v11.6.1 BIG-IP systems.
+* Failed configurations for objects should not prevent future configurations from happening.
```

Comment:
If we are describing the bugfix, I would prefer 'do not' instead of 'should not'.